### ISSUE TITLE
geminabox-release fails with https

### ISSUE THREAD

**Issue by kikito** [dfherr/geminabox-release#4]
My geminabox is on an https uri. `geminabox-release` throws an error when trying to push to it:

```
$ rake inabox:push --trace                                                                                                                                                                           
** Invoke inabox:push (first_time)
** Execute inabox:push
nixon-dns 0.0.9 built to pkg/nixon-dns-0.0.9.gem.
rake aborted!
Error (400 received)

<html>
<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>
<body bgcolor="white">
<center><h1>400 Bad Request</h1></center>
<center>The plain HTTP request was sent to HTTPS port</center>
<hr><center>openresty/1.7.0.1</center>
</body>
</html>
~/.rvm/gems/ruby-2.1.2/gems/geminabox-release-0.1.1/lib/geminabox-release.rb:138:in `inabox_push'
~/.rvm/gems/ruby-2.1.2/gems/geminabox-release-0.1.1/lib/geminabox-release.rb:91:in `push_inabox_gem'
~/.rvm/gems/ruby-2.1.2/gems/geminabox-release-0.1.1/lib/geminabox-release.rb:70:in `block in install'
~/.rvm/gems/ruby-2.1.2/gems/rake-10.4.2/lib/rake/task.rb:240:in `call'
~/.rvm/gems/ruby-2.1.2/gems/rake-10.4.2/lib/rake/task.rb:240:in `block in execute'
...
~/.rvm/gems/ruby-2.1.2/gems/rake-10.4.2/bin/rake:33:in `<top (required)>'
~/.rvm/gems/ruby-2.1.2/bin/rake:23:in `load'
~/.rvm/gems/ruby-2.1.2/bin/rake:23:in `<main>'
~/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `eval'
~/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `<main>'
```

My knowledge of `Net::HTTP` is limited, but I think what is needed is:

* requiring `net/https` in addition to `net/http`
* Detecting whether the protocol of the uri is `http` or `https`
* If it's the latter, modifying the `http` variable before doing the post like so: `http.use_ssl = true`

**Comment by dfherr** [comment]
fastly introduced this in their fork. I don't know if it's working for them (but i guess so), but they simply used:

```
if uri.scheme == 'https'
          http.use_ssl = true
          http.verify_mode = OpenSSL::SSL::VERIFY_NONE
end
```

**Comment by hilli** [comment]
Oh, there was an issue already, on my pull request...

The request works, at least, on a site with a real certificate. Which I would call better than nothing. It would be an idea to make it an option (to self.patch) for it to use :ssl_verify => false to set the  OpenSSL::SSL::VERIFY_NONE option on.

**Comment by dfherr** [comment]
added with #5 and released in 0.2.0
